Source/JavaScriptCore/wasm/WasmAirIRGenerator64.cpp

```diff
@@ -272,6 +272,8 @@ class AirIRGenerator64 : public AirIRGeneratorBase<AirIRGenerator64, TypedTmp> {
         AIR_OP_CASE(Not)
         AIR_OP_CASE(Neg)
 
+        else if (op == SIMDLaneOperation::RelaxedTruncSat) airOp = B3::Air::VectorTruncSat;
```

Comment:
You can skip this file, we don't use this tier

Reply:
Commenting out this line leads to the `wasm-air` tests failing, since OMG can't translate `RelaxedTruncSat` into any Air opcode.